src/JoomlaBrowser.php

```diff
@@ -225,6 +225,20 @@ public function checkForPhpNoticesOrWarnings($page = null)
     }
 
     /**
+     * Selects an option in a Joomla Radio Field based on its label
+     *
+     * @return void
+     */
+    public function selectOptionInRadioField($label, $option)
+    {
+        $I = $this;
+        $label = $I->findField(['xpath' => "//label[contains(normalize-space(string(.)), '$label')]"]);
+        $radioId = $label->getAttribute('for');
+
+        $I->click("//fieldset[@id='$radioId']/label[contains(normalize-space(string(.)), '$option')]");
```

Comment:
Adding a debug should increase the readability something like this
```
$this->debug("Trying to select the $option from the $label");
```
What do you think?